core/types.h

```diff
@@ -655,7 +657,6 @@ struct settings_t
 	struct
 	{
 		u32 cable;			// 0 -> VGA, 1 -> VGA, 2 -> RGB, 3 -> TV
-		u32 RTC;
```

Comment:
Is there still some way to specify RTC eg for determinism or netplay?

Reply:
not at the moment. It should probably be saved as part of the save state though.